app/src/main/java/com/mapbox/maps/testapp/examples/InsetMapActivity.kt

```diff
@@ -103,7 +103,7 @@ class InsetMapActivity : AppCompatActivity(), OnCameraChangeListener {
   }
 
   override fun onCameraChanged() {
-    val mainCameraPosition = mainMapboxMap.getCameraOptions()
+    val mainCameraPosition = mainMapboxMap.getCameraOptions(null)
```

Comment:
I feel a lot of pain seeing it as this will be replaced by `getCameraState()` very soon... Basically meaning that all those occurrences will be replaced again

Reply:
shall we wait for the gl-native adoption? I think we an also merge and adapt later.

Reply:
`I feel a lot of pain seeing it as this will be replaced by getCameraState()` 
will result in pain, with or without. :+1: on merging and adapting later.